README.md

```diff
@@ -540,3 +539,200 @@ describe('testing timeouts', () => {
   })
 })
 ```
+
+### Conditional Mocking
+
+In some test scenarios, you may want to temporarily disable (or enable) mocking for all requests or the next (or a certain number of) request(s). 
+You may want to only mock fetch requests to some URLs that match a given request path while in others you may want to mock
+all request except those matching a given request path. You may even want to conditionally mock based on request headers.
+
+The conditional mock functions cause `jest-fetch-mock` to pass fetches through to the concrete fetch implementation conditionally. 
+Calling `fetch.dontMock`, `fetch.doMock`, `fetch.onlyMockIf` or `fetch.neverMockIf` overrides the default behavior
+of mocking/not mocking all requests.  `fetch.dontMockOnce`, `fetch.doMockOnce`, `fetch.onlyMockOnceIf` and `fetch.neverMockOnceIf` only overrides the behavior 
+for the next call to `fetch`, then returns to the default behavior (either mocking all requests or mocking the requests based on the last call to
+`fetch.dontMock`, `fetch.doMock`, `fetch.onlyMock` and `fetch.neverMock`).
+
+Calling `fetch.resetMocks()` will return to the default behavior of mocking all fetches with a text response of empty string.
+- `fetch.dontMock()` - Change the default behavior to not mock any fetches until `fetch.resetMocks()` or `fetch.doMock()` is called
```

Comment:
I choose to use "dont/do" for the functions that complete disable/enable mocking (with no conditions) and "only if/never if" for the conditional functions to make it clear which were which.